documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,137 @@
+
+[Sachin Kumar](https://wiki.camaraproject.org/display/~sachinvodafone)
+
+DAVID VALLEJO NAVARRO
+
+[Diego González Martínez](https://wiki.camaraproject.org/display/~diegogonmar)
+
+[Jan Friman](https://wiki.camaraproject.org/display/~Jan_Friman)
+
+Juan Antonio Hernando La...
+
+Juan Fabio Garcia Solero
+
+PEDRO BALLESTEROS HERRANZ
+
+[Ramesh Shanmugasundaram](https://wiki.camaraproject.org/display/~sfnuser)
+
+<br>
+
+## Agenda
+
+1. Recent updates & recap
+2. Issues and PRs. Priority discussions (most active issues and/or dependencies for release v0.2):
+    - **Camara profile of OIDF and IETF standards**
+        - Please comment on https://github.com/camaraproject/IdentityAndConsentManagement/issues/122 .
+    - **offline access** #123 https://github.com/camaraproject/IdentityAndConsentManagement/pull/123
+    - **Discussion on Issue #100 "Claims to be used in oAuth Access tokens".**
+    - **Discussion on Issue #108 "Question to documentation update for security schemes".**
+    - **Discussion on Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".**
+
+3. Action Points
+4. AoB
+
+<br>
+
+## Meeting Technicallities
+
+Jesús offered that ownership of the meeting invite could be transferred to the new chair. Axel said that for now this can stay as it is.
+Jesús points out that the current owner of the meeting invitation (Jose Antonio Ordoñez Lucena) is leaving the company, and it might make sense for Axel, as moderator, to have his own meeting invitation under his control. Because the meeting invitation may be left uncontrolled (no possibility to change the schedule, etc.) once Jose Antonio leaves and we may need to create a new meeting invitation anyway.
+
+
+## Discussion on Consensus in Camara Working Groups
+
+Recapitulation of the previous discussion on this topic: Jesús and Diego said there was consensus achieved during a WG call that we continue working on https://github.com/camaraproject/IdentityAndConsentManagement/pull/113
+Herbert said that lazy-consensus is the Camara way. Shilpa said no consensus was achieved. 
+Jesús disagrees and clarifies from his view of what happened in the previous call that the WG participants were asked, considering the different opinions, if we could agree as a way forward to continue with the "self-contained" contribution (PR #113) with compromise to properly note any changes related to the reused standard. And there was support from Ericsson, GSMA, and no objection from others on the call. DT of course preferred the delta approach, but Axel mentioned that he wouldn't disagree with the WG as long as the changes to the standard are properly marked in the document.
+Axel proposed that consensus should be achieved on Github because not everybody can attend meetings. Diego said that is not the Camara way and all CAMARA subprojects actually make decisions during the calls, and active WG participants are well aware of this.
```

Comment:
Can we make clearer that proposal to not to allow decisions to be taken in meetings was dismissed? Otherwise it seems that there is an open discussion on this, which I think it is not.
This is compatible with following part of the minutes commenting about enhancing decision-making or with communication about decision to be taken in advance.

Reply:
added that my proposal was dismissed and that lazy-consensus it is in Camara WGs